inst/qml/Upgrades.qml

```diff
@@ -0,0 +1,148 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:	"ClassicalMetaAnalysis"
+		fromVersion:	"0.15"
```

Comment:
```suggestion
		fromVersion:	"0.16.4"
```

Reply:
Hi @Kucharssim, thank you for the suggestions. I have updated the files accordingly.